torchinductor/ir.py

```diff
@@ -2556,26 +2644,35 @@ def map_args(self):
                     if len(in_args) >= 3
                     else "None",
                 ),
-                ("delta_x_ptr", "None"),
+                # ("delta_x_ptr", "None"),
                 ("BATCH", f"{self.inputs[0].get_size()[0]}"),
                 ("IN_C", f"{self.inputs[0].get_size()[1]}"),
                 ("IN_H", f"{self.inputs[0].get_size()[2]}"),
                 ("IN_W", f"{self.inputs[0].get_size()[3]}"),
-                ("KERNEL_N", f"{self.inputs[1].get_size()[0]}"),
-                ("KERNEL_H", f"{self.inputs[1].get_size()[2]}"),
-                ("KERNEL_W", f"{self.inputs[1].get_size()[3]}"),
+                (
+                    "KERNEL_N",
+                    f"{V.graph.sizevars.size_hint(self.inputs[1].get_size()[0])}",
+                ),
+                (
+                    "KERNEL_H",
+                    f"{V.graph.sizevars.size_hint(self.inputs[1].get_size()[2])}",
+                ),
+                (
+                    "KERNEL_W",
+                    f"{V.graph.sizevars.size_hint(self.inputs[1].get_size()[3])}",
+                ),
                 ("OUT_H", f"{self.get_size()[2]}"),
                 ("OUT_W", f"{self.get_size()[3]}"),
-                ("stride_h", f"{const_args[0][0]}"),
-                ("stride_w", f"{const_args[0][1]}"),
-                ("padding_h", f"{const_args[1][0]}"),
-                ("padding_w", f"{const_args[1][1]}"),
-                ("dilation_h", f"{const_args[2][0]}"),
-                ("dilation_w", f"{const_args[2][1]}"),
+                ("stride_h", f"{V.graph.sizevars.size_hint(const_args[0][0])}"),
```

Comment:
`size_hint` is not safe to use.   It is just a hint and could change.

Reply:
Use const_args instead